wiki/Tournaments/o!mLN/4/en.md

```diff
@@ -0,0 +1,387 @@
+
+### Additional rules/information
+
+#### Mod configurations
+
+1. The score system used for all matches in this tournament will be **Score V2**.
+2. All maps are played with FreeMod. Visual mods such as Mirror, Hidden, Fade-In, Flashlight, and any combination of these mods are allowed. You are also allowed, but not forced, to play with NoFail.
+
+#### Punctuality
+
+1. Players are required to arrive for their match before the stipulated time for their match. Otherwise, a **10 minute** grace timer will be applied by the appointed referee for the match.
+2. If the player fails to arrive at their match before this timer runs out, the match will be considered as a win by default to the other player.
+3. If the player arrives at their match during this grace period, the time remaining on the timer will be brought forward to the ban/pick proceeding of the match.
+
+#### Ban/Pick procedures
+
+1. Players will be given 2 minutes to roll and perform their first ban/picks. Subsequently, players will have 2 minutes to pick a map and get ready to play the map.
+   - If a player fails to adhere to the timer to ban/pick, the referee will choose a map by random to be picked in place of the player, using the !roll command.
+   - On the second and subsequent occurrences, a point will be automatically awarded to the other player.
+
+#### Disconnection
+
+1. If a player is disconnected from a map within the first 30 seconds of gameplay, they can ask for the map to be aborted and replayed.
+   - This is only available once per match for each player. Subsequent disconnects will give a point to the opponent and use up the remainder of the 10 minute grace period.
+2. In the event a player is unable to return to the lobby before the grace period ends, the match will be considered a win to the other player.
+3. In the event both players are disconnected, the player whose grace period runs out first will concede the match to the opponent.
+4. In the event both players run out of grace period simultaneously, the winner will be the player with the higher qualifier seed.
+
+#### General conduct
+
+1. Every staff and player must respect each other and follow the osu! community rules, no discussion about sensitive topics (for example religion, politics), and anything that could be offensive to others.
+2. Keep the competition and atmosphere friendly amongst each other, whether it is in the match, in the Twitch chatroom, or on the Discord server. Tournaments are always for us to come together as a community and not engage in pointless arguments.
+3. In the event a player or staff member is found to have conducted themselves inappropriately towards anyone in the server, said individual can be subject to timeouts, warnings and bans from the tournament and its affiliated twitch channel based on the severity of the incident.
```

Comment:
Is this only making reference to the Discord server, or the entire tournament? The point above seems to contradict that. Just want to clarify as right now this would technically only apply to the people in the server, and off-platform behaviour would not be punished; according to the current wording, that is.
```suggestion
3. In the event a player or staff member is found to have conducted themselves inappropriately towards anyone in the server, said individual can be subject to timeouts, warnings, and bans from the tournament and its affiliated Twitch channel based on the severity of the incident.
```